lib/types/index.js

```diff
@@ -14,6 +14,7 @@ exports.Embedded = require('./arraySubdocument');
 exports.DocumentArray = require('./documentArray');
 exports.Decimal128 = require('./decimal128');
 exports.ObjectId = require('./objectid');
+exports.Int32 = require('./int32');
```

Comment:
Unlike other schema types here, we don't need a type (I don't think).  The types are mongoose-specific data types.  Notice that not all schema types have associated mongoose types (String or boolean, for example).  We can remove everything related to a mongoose Type from the PR.

Reply:
No - but mongoose has specific wrappers for them in lib/types.  see https://github.com/Automattic/mongoose/issues/1671